tony-history-server/gradlew.bat

```diff
@@ -0,0 +1,84 @@
+@if "%DEBUG%" == "" @echo off

```

Comment:
You shouldn't need to check this in. We should only have one gradlew checked in in the root directory.